### ISSUE TITLE
Add support for multi:softmax objective

### ISSUE THREAD

**Issue by eafpres** [mwburke/xgboost-python-deploy#4]
I was hoping to use this in a case where I am using the multi:softmax objective:  
```
model_params = {'colsample_bytree' : colsample_bytree,
                'gamma' : gamma,
                'learning_rate' : learning_rate,
                'max_depth' : max_depth,
                'min_child_weight' : min_child_weight,
                'n_estimators' : n_iters,
                'reg_alpha' : reg_alpha,
                'reg_lambda' : reg_lambda,
                'subsample' : subsample,
                'objective' : 'multi:softmax',
                'booster' : 'gbtree',
                'eval_metric' : 'merror',  
                'num_class' : 3,
                'base_score' : 0.5}
```

I realize this may be out of scope.
Thank you for the module, still very useful.
